ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseField.json

```diff
@@ -2128,6 +2128,8 @@
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "lastEvidenceAddedDate", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "registrarDirections", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
   {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "registrarDirectionToAdd", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "transferToState", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
+  {"CaseTypeID": "GrantOfRepresentation", "CaseFieldID": "currentApplicationStateName", "UserRole": "caseworker-probate-superuser", "CRUD": "CRUD"},
```

Comment:
I believe you can use a meta data field to tell the user the current state - we dont need to save a new field
contact me!

Reply:
use CCD MetaDate instead